hack/run_test

```diff
@@ -259,7 +263,7 @@ function run_tests() {
 function run_slave() {
   local master_ip="$1"; shift
   local suffix="$1"; shift
-  docker run $cluster_args -e POSTGRESQL_MASTER_IP=${master_ip} \
+  docker run $cluster_args -p 5432 -e POSTGRESQL_MASTER_IP=${master_ip} \
```

Comment:
Why is this needed? (missing comment probably).

Can we connect the master/slave containers without exposing ports from host system?